docs/src/pages/customization/how-to-customize/how-to-customize.md

```diff
@@ -159,3 +160,50 @@ Here are four alternatives; each has its pros and cons.
 In order to promote consistency between components, and manage the user interface appearance as a whole, Material-UI provides a mechanism to apply global changes.
 
 Please take a look at the theme's [global overrides page](/customization/theme-components/) for more details.
+
+## 5. Global CSS override
```

Comment:
I think the link at the bottom of the CssBaseline page needs to be updated?

Reply:
Oops I forgot to check if the url was used and replace it. I’ll replace all occurrences.